src/components/BaseHead.astro

```diff
@@ -52,7 +52,7 @@ const resolvedImage = {
 
 <link
 	rel="preload"
-	href="/fonts/MD IO/Web/MDIO0.5-Regular.woff2"
+	href="/fonts/MDIO/Web/MDIO0.6-Regular.woff2"
```

Comment:
Thank you for this, I kept getting annoyed that this font was wrong and I couldn't figure out why 🫠